templates/includes/record-embed--no-image.html

```diff
@@ -0,0 +1,25 @@
+<div class="record-embed-no-image">
+    <img class="record-embed-no-image__icon" src="/static/images/fontawesomesvgs/search-white.svg" alt="">
+    <p class="record-embed-no-image__icon-label" role="presentation" aria-hidden="true">
+        Discover our records
+    </p>
+
+    <h3 class="record-embed-no-image__heading">
+        Shakespeare and James I
+        <span class="sr-only">Discover our records</span>
```

Comment:
Similar to other comments, I wonder if 'Discover our records' should appear before the heading text, possibly followed by a full stop or semi-colon.

> Discover our records. Shakespeare and James I

rather than

> Shakespeare and James I Discover our records